simplesat/dependency_solver.py

```diff
@@ -142,6 +143,82 @@ def requirements_are_satisfiable(packages, requirements, modifiers=None):
         return False
 
 
+def satisfy_requirements(packages, requirements, modifiers=None):
+    """ Find a collection of packages that satisfy the requirements.
+
+    Parameters
+    ----------
+    packages : iterable of PackageMetadata
+        The packages available to draw from when satisfying requirements.
+    requirements : list of Requirement
+        The requirements used to identify relevent packages.
+    modifiers : ConstraintModifiers, optional
+        If not None, modify requirements before resolving packages.
+
+    Returns
+    -------
+    tuple of PackageMetadata
+        Return a list of packages that together satisfy all of the
+        `requirements`.
+
+    Raises
+    ------
+    SatisfiabilityError
+        If the `requirements` cannot be satisfied using the `packages`.
+
+    MissingInstallRequires
```

Comment:
This failure mode is actually only possible when the solver is told to be "strict." That feature is meant for packages developers to have a sanity check before release.

I'll fix the doc and update the test.